examples/framework-multiple/src/components/VueCounter.vue

```diff
@@ -9,7 +9,7 @@
 	</div>
 </template>
 
-<script>
+<script lang="ts">
```

Comment:
Unfortunately, due to how Volar is setup, `@types/react` is used for type checking the template instead of `@vue/jsx-runtime` and this file has errors over the `class` attributes. There's no way to fix this